tests/integration/update_cluster/ha_gce/kubernetes.tf

```diff
@@ -17,555 +17,341 @@ output "region" {
-  name               = "a-nodes-ha-gce-example-com"
-  zone               = "us-test1-a"
   base_instance_name = "nodes"
-
-  version = {
-    instance_template = "${google_compute_instance_template.nodes-ha-gce-example-com.self_link}"
+  name               = "a-nodes-ha-gce-example-com"
+  target_size        = 1
+  version {
+    instance_template = google_compute_instance_template.nodes-ha-gce-example-com.self_link
   }
-
-  target_size = 1
+  zone = "us-test1-a"
 }
 
 resource "google_compute_instance_group_manager" "b-master-us-test1-b-ha-gce-example-com" {
-  name               = "b-master-us-test1-b-ha-gce-example-com"
-  zone               = "us-test1-b"
   base_instance_name = "master-us-test1-b"
-
-  version = {
-    instance_template = "${google_compute_instance_template.master-us-test1-b-ha-gce-example-com.self_link}"
+  name               = "b-master-us-test1-b-ha-gce-example-com"
+  target_size        = 1
+  version {
+    instance_template = google_compute_instance_template.master-us-test1-b-ha-gce-example-com.self_link
   }
-
-  target_size = 1
+  zone = "us-test1-b"
 }
 
 resource "google_compute_instance_group_manager" "b-nodes-ha-gce-example-com" {
-  name               = "b-nodes-ha-gce-example-com"
-  zone               = "us-test1-b"
   base_instance_name = "nodes"
-
-  version = {
-    instance_template = "${google_compute_instance_template.nodes-ha-gce-example-com.self_link}"
+  name               = "b-nodes-ha-gce-example-com"
+  target_size        = 1
+  version {
+    instance_template = google_compute_instance_template.nodes-ha-gce-example-com.self_link
   }
-
-  target_size = 1
+  zone = "us-test1-b"
 }
 
 resource "google_compute_instance_group_manager" "c-master-us-test1-c-ha-gce-example-com" {
-  name               = "c-master-us-test1-c-ha-gce-example-com"
-  zone               = "us-test1-c"
   base_instance_name = "master-us-test1-c"
-
-  version = {
-    instance_template = "${google_compute_instance_template.master-us-test1-c-ha-gce-example-com.self_link}"
+  name               = "c-master-us-test1-c-ha-gce-example-com"
+  target_size        = 1
+  version {
+    instance_template = google_compute_instance_template.master-us-test1-c-ha-gce-example-com.self_link
   }
-
-  target_size = 1
+  zone = "us-test1-c"
 }
 
 resource "google_compute_instance_group_manager" "c-nodes-ha-gce-example-com" {
-  name               = "c-nodes-ha-gce-example-com"
-  zone               = "us-test1-c"
   base_instance_name = "nodes"
-
-  version = {
-    instance_template = "${google_compute_instance_template.nodes-ha-gce-example-com.self_link}"
+  name               = "c-nodes-ha-gce-example-com"
+  target_size        = 0
+  version {
+    instance_template = google_compute_instance_template.nodes-ha-gce-example-com.self_link
   }
-
-  target_size = 0
+  zone = "us-test1-c"
 }
 
 resource "google_compute_instance_template" "master-us-test1-a-ha-gce-example-com" {
-  can_ip_forward = true
```

Comment:
it looks like `google_compute_instance_template` isn't getting rendered properly. My guess is because it uses type embedding, i'll investigate further.

https://github.com/kubernetes/kops/blob/3344ff46bbdc9ebcb2675a20ba6d30a416bd3891/upup/pkg/fi/cloudup/gcetasks/instancetemplate.go#L419-L422

/hold